### PR TITLE
Fix Rectangle "Resized" function

### PR DESCRIPTION
**Pull request by PlainSight**
- Fixed order of operations in Resized function which essentially made all resizes anchor at the origin
- Added some simple test cases

### CODE REVIEW

File: geometry_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	answers := []pixel.Rect{
+		pixel.R(-5, -5, 5, 5),
+		pixel.R(15, 15, 25, 25),
+	}
+
+	for i, rect := range testCases {
+		answer := answers[i]
+
+		// resize rectangle by 50% anchored at it's current center point
+		resizedRect := rect.Resized(rect.Center(), rect.Size().Scaled(0.5))
```
Comment (faiface) [inline geometry_test.go]: test different anchors, at least Min, Max and Center

**Comment by faiface** [comment]
Cool! I'll merge. The tests could be a little more DRY (by putting those things in a loop somehow), but I don't want to bother you too much. If you want to, you can make it more DRY, but if not, I'll do it later. Thanks!

**Comment by PlainSight** [comment]
I actually wrote it in a less repetitive way to start but decided to duplicate things out as I though it reduced visibility on which tests were failing. I think I have a better solution though which I might push later. I might include a couple of tests for some other geo functions if I do.

**Comment by faiface** [comment]
Doing a slice of subtests with names and then running them in a loop by `t.Run` could fix the visibility problem. Doing more tests will be appreciated!

**Pull request merged**